src/instructlab/sdg/utils.py

```diff
@@ -304,3 +316,15 @@ def chunk_document(documents: List, server_ctx_size, chunk_word_count) -> List[s
         content.extend([item.page_content for item in temp])
 
     return content
+
+
+def get_model_family(forced, model_path):
```

Comment:
should add a TODO comment that this code is copied from the CLI.  The CLI could use it from here so it's not in 2 copies.